perf/perftool.py

```diff
@@ -51,6 +59,19 @@ def setUp(self):
         self.srcdir = os.path.join(self.srcdir, 'perftool-testsuite-master')
 
     def test(self):
+        '''
+        perf test :Does sanity tests and
+        execute the tests by calling each module
+        '''
+        # Built in perf test
+        result = process.run("perf test")
+        x = result.stderr.splitlines()
+        for string in x:
```

Comment:
can this be like 
```
    for failedString in process.run("perf test").stderr.splitlines():
           if 'FAILED' in str(failedString):
                   self.fail("Test case failed is %s", y.strip("[]")
```

just eliminate extra variables and self.log to self.fail as i think in this check we need to fail the test